### ISSUE TITLE
Add Chat related options

### ISSUE THREAD

**Issue by u8Jj3** [albertobeloni/CUI#3]
Hello,

This addon is really great for immersion. The only downside for me is the chat.
I feel like there is some missing options to the chat module for it to be functional.

If this is possible, here is some QoL options that would be really great to have in CUI.

### Chat module
- Add  an option to fade in the chat when in combat, fade out when out of combat.
- Add an option to fade in the chat when in a Instance/raid.
- Add an option to fade in the chat when there is a new message in the chat.
  - With configurable per channel filter (For exemple I want the chat to fade in for whisper, party, raid messages but not /1, /2, ...
  - With configurable fade in/out delay in seconds when showing new messages.

In other words, a condition/variable system for the chat would be great but I dunno if this is possible.

If this is not possible for you to add theses options, would you consider atleast add an option to disable the chat fade out ? For us to use another chat addon without conflicting with CUI ? 

Thanks! :)

**Comment by albertobeloni** [comment]
The option to disable the chat fading is easy enough, I'll see if I can release an update today or tomorrow. The other suggestions are great but quite a bit more complex. Not discarding adding some of those in the future but probably not in the near future.

**Comment by u8Jj3** [comment]
> The option to disable the chat fading is easy enough, I'll see if I can release an update today or tomorrow. The other suggestions are great but quite a bit more complex. Not discarding adding some of those in the future but probably not in the near future.

Thanks! Glad to hear that :))
